zerver/openapi/zulip.yaml

```diff
@@ -8420,6 +8420,29 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: delete-custom-emoji
+      summary: Delete custom emoji
```

Comment:
Let's change the naming to be `deactivate-custom-emoji` -- that's how we explain the feature in the UI. https://zulip.com/help/custom-emoji#deactivate-custom-emoji. Similarly the text should all talk about "deactivate a custom emoji", etc.